docs/content/doc/developers/guidelines-backend.md

```diff
@@ -67,7 +67,7 @@ Some actions should allow for rollback when database record insertion/update/del
 So services must be allowed to create a database transaction. Here is some example,
 
 ```go
-// servcies/repository/repo.go
+// services/repository/repository.go
 func CreateXXXX() error {\
   ctx, committer, err := db.TxContext()
```

Comment:
This code block can now be updated to use `db.WithTx(…)`
Should this be done in this PR as well.